services/metadata/src/metadata/routes/domain/permanent_id_requests.clj

```diff
@@ -0,0 +1,80 @@
+(ns metadata.routes.domain.permanent-id-requests
+  (:use [common-swagger-api.schema :only [describe
+                                          PagingParams
+                                          SortFieldDocs
+                                          SortFieldOptionalKey
+                                          StandardUserQueryParams]]
+        [clojure-commons.error-codes]
+        [metadata.routes.domain.common])
+  (:require [metadata.persistence.permanent-id-requests :as db]
+            [schema.core :as s])
+  (:import [java.util UUID]))
+
+(def PermanentIDRequestIdParam (describe UUID "The Permanent ID Requests's UUID"))
+(def PermanentIDRequestTypeEnum (apply s/enum (map :type (db/list-permanent-id-request-types))))
```

Comment:
It does, but we also already do this elsewhere: https://github.com/cyverse/DE/blob/dev/services/metadata/src/metadata/routes/domain/template.clj#L9